components/cf-app-ssh/backend/cf_websocket_app_ssh.go

```diff
@@ -96,19 +96,14 @@ func (cfAppSsh *CFAppSsh) appSSH(c echo.Context) error {
 	// cf:APP-GUID/APP-INSTANCE-INDEX@SSH-ENDPOINT
 	username := fmt.Sprintf("cf:%s/%s@%s", appGUID, appInstance, host)
 
-	clientID, err := p.GetClientId("cf")
-	if err != nil {
```

Comment:
This is required to make SSH work when Stratos is pushed as a CF app, since we use the CF client ID and secret.